src/DittoMachine.sol

```diff
@@ -186,38 +202,52 @@ contract DittoMachine is ERC721, ERC721TokenReceiver, ERC1155TokenReceiver {
             _tokenId = FLOOR_ID;
         }
 
-        // calculate cloneId by hashing identifiying information
-        uint256 cloneId = uint256(keccak256(abi.encodePacked(
+        // calculate protoId by hashing identifiying information, precursor to cloneId
+        uint256 protoId = uint256(keccak256(abi.encodePacked(
             _ERC721Contract,
             _tokenId,
             _ERC20Contract,
             floor
         )));
+        // hash protoId and index to get cloneId
+        uint256 cloneId = uint256(keccak256(abi.encodePacked(protoId, index)));
 
-        _updatePrice(cloneId);
+        _updatePrice(protoId);
 
         uint256 value;
         uint256 subsidy;
+        uint256 elderId;
```

Comment:
can we move this declaration just before its use.

Reply:
probably can, i think i had it not being assigned in nested if statements before and this is a remnant